Packs/PrismaCloudCompute/Playbooks/Prisma_Cloud_Compute_-_Get_Container_Events.yml

```diff
@@ -0,0 +1,331 @@
+id: Prisma Cloud Compute - Get Container Events
+version: -1
+name: Prisma Cloud Compute - Get Container Events
```

Comment:
I think we should work with playbook outputs and not with set and map actions inside the playbook. It will make the use of the playbook easier to the next people who will use it.